RecoLocalCalo/HcalRecAlgos/src/HybridMinimizer.cc

```diff
@@ -324,8 +324,8 @@ bool HybridMinimizer::Minimize() {
       // print the real number of maxfcn used (defined in ModularFuncitonMinimizer)
       int maxfcn_used = maxfcn; 
       if (maxfcn_used == 0) { 
-         int nvar = fState.VariableParameters();
-         maxfcn_used = 200 + 100*nvar + 5*nvar*nvar;
+	//         int nvar = fState.VariableParameters();
```

Comment:
Please remove the unneeded commented out code (as well as the "cout" below)

Reply:
Indeed, the whole chunk of code under ```if (PrintLevel() >=1)``` is unused. Please decide to wholly remove it, or comment out everything. Commenting out unused must normally be avoided: however this whole class is full of commented out ```cout```s, and it would require a major cleaning, which could be probably get deferred to a later PR (unless you want to do it now here),

Reply:
@perrotta 
yes exaclty this PR has huge block everywhere commented
https://github.com/cms-sw/cmssw/blob/master/RecoLocalCalo/HcalRecAlgos/src/HybridMinimizer.cc#L727
https://github.com/cms-sw/cmssw/blob/master/RecoLocalCalo/HcalRecAlgos/src/HybridMinimizer.cc#L482
https://github.com/cms-sw/cmssw/blob/master/RecoLocalCalo/HcalRecAlgos/src/HybridMinimizer.cc#L417
those are in since the 2015 when the M2 was used to debug.

I initially didn't propose to act on the HybridMinimizer.cc but you asked to comments lines that create warning unused variable in the compilation. This is is what I did.

At this stage I'm not going to delete commented lines (and do partial job), because in case someone has to debug events those lines can be uncommented and become useful.

The proposal here is take the PR as is now: i.e. remove the "warning as compilation" or I will just reset the HybridMinimizer.cc to what is the release.  Later in the year a more cleanup of obsolete files (this files is only used by M2, algo  that is being dismissed).

Please let me know what you prefer.

Reply:
@mariadalfonso : I think that the "minimal proposal" I made may correspond to what you also propose:
- fully remove line 193, as it is not needed to any debug
- comment out everything from line 323 to line 333 (why only lines 327-328?)
- comment out everything from line 704 to line 713 (why only lines 708-709?)

As I wrote, I understand that removing or modifying all those cout's is out of the scope of ths PR, therefore applying only the minimal and non-invasive adjustments above is fine with me here.

Reply:
@perrotta 
ok, I will comment out everything from line 323 to line 333, 704 to line 713 (this was as last saturda :( ) + remove line 193

Reply:
if the code is needed for debugging, just add an appropriate switch to enable/disable it (it can be a configuration flag or a compiler macro). With a config switch you get some assurance of future maintenance of that code.